src/components/bs5/header/header.data.json

```diff
@@ -163,9 +163,7 @@
             "placeholder": "",
             "label": "",
             "formAction": {
-                "url": "https://www.forgov.qld.gov.au/search",
-                "suggestions": "https://discover.search.qld.gov.au/s/suggest.json",
-                "results": "https://discover.search.qld.gov.au/s/search.json"
+                "url": "https://www.forgov.qld.gov.au/search"
```

Comment:
This is old meta data not in use by this component (is now in search input component)